### PR TITLE
[GLT-2906] DB Migration, updates for new col names

### PR DESCRIPTION
**Pull request by avarsava**

### CODE REVIEW

File: sqlstore/src/main/resources/db/migration_beforeMigrate/05_add_box_parameters.sql [hunk 1]
```diff
@@ -8,9 +8,9 @@ CREATE PROCEDURE addBoxSize(
   iColumns int,
   iScannable tinyint(1)
 ) BEGIN
-  IF NOT EXISTS (SELECT 1 FROM BoxSize WHERE `rows` = iRows AND `columns` = iColumns AND scannable = iScannable)
+  IF NOT EXISTS (SELECT 1 FROM BoxSize WHERE `boxSizeRows` = iRows AND `boxSizeColumns` = iColumns AND scannable = iScannable)
   THEN
-    INSERT INTO BoxSize(`rows`, `columns`, scannable)
+    INSERT INTO BoxSize(`boxSizeRows`, `bSizeColumns`, scannable)
```
Comment (djcooke) [inline sqlstore/src/main/resources/db/migration_beforeMigrate/05_add_box_parameters.sql]: `bSizeColumns` -> `boxSizeColumns`

File: core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/BoxSize.java [hunk 2]
```diff
@@ -25,8 +25,10 @@
   @GeneratedValue(strategy = GenerationType.AUTO)
   @Column(name = "boxSizeId")
   private long id = UNSAVED_ID;
-  private Integer rows;
-  private Integer columns;
+  @Column(name = "boxSizeRows")
+  private Integer boxSizeRows;
```
Comment (djcooke) [inline core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/BoxSize.java]: `@Column` annotation isn't necessary if the Java field name matches the DB field name (though it also does no harm)

File: sqlstore/src/main/resources/db/migration/V8000__fix_box_size.sql [hunk 3]
```diff
@@ -0,0 +1,28 @@
+-- Rename old table
+ALTER TABLE `BoxSize`
+    RENAME TO `BoxSize_old`;
+
+-- Recreate table
+CREATE TABLE `BoxSize` (
+  `boxSizeId` bigint(20) NOT NULL AUTO_INCREMENT,
+  `boxSizeRows` bigint(20) NOT NULL,
+  `boxSizeColumns` bigint(20) NOT NULL,
+  `scannable` boolean DEFAULT 0,
```
Comment (djcooke) [inline sqlstore/src/main/resources/db/migration/V8000__fix_box_size.sql]: Shouldn't be nullable (though I realize it was in the existing table)

**Comment by djcooke** [comment]
Please include the ticket number in commit message

File: sqlstore/src/main/resources/db/migration/V8000__fix_box_size.sql (hunk 3 above)
Comment (avarsava) [reply sqlstore/src/main/resources/db/migration/V8000__fix_box_size.sql]: Fixed

File: sqlstore/src/main/resources/db/migration_beforeMigrate/05_add_box_parameters.sql (hunk 1 above)
Comment (avarsava) [reply sqlstore/src/main/resources/db/migration_beforeMigrate/05_add_box_parameters.sql]: Fixed

**Pull request merged**